src/components/Viewport.ts

```diff
@@ -473,16 +482,35 @@ export default class Viewport {
   }
 
   public destroy(): void {
+    const state = this.state;
+    const wrapper = this.flicking.getElement();
     const viewportElement = this.viewportElement;
-    const wrapper = viewportElement.parentElement;
+    const cameraElement = this.cameraElement;
+
+    const originalPanels = this.panelManager.originalPanels();
+
+    restoreStyle(viewportElement, state.originalViewportStyle);
```

Comment:
@WoodNeck 
Do you think the condition to call restoreStyle function also requires `isCameraGiven` and `isViewportGiven`?

Reply:
Well that would be more efficient, but I think that's still trivial.